pom.xml

```diff
@@ -50,10 +50,10 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <!--<jenkins.version>2.46.3</jenkins.version>-->
-        <java.level>6</java.level>
-        <!--<java.level>7</java.level>-->
+        <jenkins.version>2.60.2</jenkins.version>
+        <jenkins-test-harness.version>2.23</jenkins-test-harness.version>
+        <java.level>8</java.level>
```

Comment:
I like Java 8 as well but old version of Jenkins could be runned on any Java so we don't want to stop support them.